.travis.yml

```diff
@@ -1,16 +1,9 @@
 language: ruby
 matrix:
   allow_failures:
-    - rvm: jruby-head
     - rvm: ruby-head
 rvm:
-  - jruby-19mode
-  - jruby-head
-  - 1.9.3
-  - 2.0.0
-  - 2.1.5
-  - 2.1.6
-  - 2.2.2
```

Comment:
We originally wanted to make sure it worked across a variety of ruby versions because this is an open source gem and others may want to use it.

Was something special done that is definitely not supported in older ruby versions?